### ISSUE TITLE
[AI] TSMixer baseline training + ONNX export

### ISSUE THREAD

**Issue by JasonEran** [JasonEran/Aether-Guard#35]
Epic: #11

## Goal
Train a lightweight time-series model and export for agent inference.

## Acceptance Criteria
- [ ] Training script reproducible.
- [ ] ONNX export validated.

**Comment by JasonEran** [comment]
#35 ??????(2026-02-24 UTC)

??????:43cc805 (eat(v2.3): add TSMixer baseline training and ONNX export workflow)

## ????
- ????:scripts/model_training/train_tsmixer_baseline.py
- ??????:scripts/model_training/README.md
- ????:scripts/model_training/requirements.txt
- M2 TSMixer ??:docs/AI-TSMixer-Baseline-v2.3-M2.md
- README ????:README.md

## ????
- ????:python -m pip install -r scripts/model_training/requirements.txt
- ????:python -m py_compile scripts/model_training/train_tsmixer_baseline.py
- ????(????):
  - python scripts/model_training/train_tsmixer_baseline.py --epochs 6 --batch-size 128 --output-dir .tmp/tsmixer-baseline-smoke2
  - ??:	rain_acc=0.7216, al_acc=0.7301, 	est_acc=0.6715
  - ONNX ??:checker=True, parity=True, max_abs_diff=8.523e-06
- ???????(? #34 spot CSV):
  - python scripts/model_training/train_tsmixer_baseline.py --dataset-csv <spot_csv> --epochs 4 --batch-size 128 --output-dir .tmp/tsmixer-baseline-from-spot2
  - ?? spot ??????????? synthetic,?? 	raining_summary.json ?? allback_reason
  - ONNX ??:checker=True, parity=True, max_abs_diff=6.020e-06

## ????
- [x] Training script reproducible
  - ?? seed + deterministic ?? + ??/???? 	raining_summary.json
- [x] ONNX export validated
  - onnx.checker ??
  - onnxruntime ? PyTorch ????(max abs diff < 1e-4)

?? #35 ????
